Packs/Venafi/Integrations/VenafiV2/VenafiV2.py

```diff
@@ -0,0 +1,371 @@
+        human_readable.append(certificate_details)
+
+    markdown_table = tableToMarkdown('Venafi certificates', human_readable)
+
+    return CommandResults(
+        outputs_prefix=CONTEXT_OUTPUT_BASE_PATH,
+        outputs=outputs,
+        raw_response=response,
+        readable_output=markdown_table
+    )
+
+
+def get_certificate_details_command(client: Client, args: dict[str, Any]) -> CommandResults:
+    """
+    Get certificate details.
+
+    Args:
+        client (Client): A Venafi client.
+        args (dict): The arguments for the command passed to the request.
+    Returns:
+        A CommandResult object with an outputs, raw_response and readable table, in case of a successful action.
+    """
+
+    outputs: dict[str, Any] = {}
+    guid = args.get('guid', '')
+    response = client._get_certificate_details(guid)
+    if response:
+        outputs = response
+
+    human_readable = []
+    certificate_details = get_human_readable_object(outputs)
+    human_readable.append(certificate_details)
+
+    markdown_table = tableToMarkdown('Venafi certificate details', human_readable)
+
+    return CommandResults(
+        outputs_prefix=CONTEXT_OUTPUT_BASE_PATH,
+        outputs=outputs,
+        raw_response=response,
+        readable_output=markdown_table
+    )
+
+
+''' HELPER FUNCTIONS '''
+
+
+def get_human_readable_object(certificate: dict[str, Any]) -> Dict[str, Any]:
+    """
+    Create a human readable object.
+
+    Args:
+        certificate (dict): An object that contains the certificate information.
+    Returns:
+        certificate_details (dict): Certificate details dictionary.
+    """
+
+    certificate_guid = certificate.get('Guid', '')
+    certificate_id = certificate_guid[1:-1]  # Guid represented as {guid}
+    certificate_details = {
+        'CreatedOn': certificate.get('CreatedOn'),
+        'DN': certificate.get('DN'),
+        'Name': certificate.get('Name'),
+        'ParentDN': certificate.get('ParentDn'),
+        'SchemaClass': certificate.get('SchemaClass'),
+        'ID': certificate_id
+    }
+
+    return certificate_details
+
+
+def delete_links_from_response(response: dict[str, Any]) -> dict[str, Any]:
+    """
+    Delete links list from the response
+
+    Args:
+        response (dict): raw response
+    Returns:
+        response (dict): response without the links list
+    """
+
+    certificates = response.get('Certificates', [])
+    for certificate in certificates:
+        if certificate.get('_links'):
+            del certificate['_links']
+
+    return response
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:  # pragma: no cover
+    """
+    main function, parses params and runs command functions
+
+    :return:
+    :rtype:
+    """
+
+    demisto_params = demisto.params()
+    base_url = demisto_params.get('server', 'https://ao-tlspd.dev.ven-eco.com')
+    username = demisto_params.get('credentials')['identifier']
```

Comment:
We always use `.get` and not brackets. Lets change it to using get.
Same as for password.

```suggestion
    username = demisto_params.get('credentials', {}).get('identifier')
```